input/postgres/buffer_cache.go

```diff
@@ -0,0 +1,56 @@
+package postgres
+
+import (
+	"context"
+
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+)
+
+// See also https://www.postgresql.org/docs/current/pgbuffercache.html
+const bufferCacheSQL string = `
+SELECT reldatabase, relfilenode, count(*) * current_setting('block_size')::int
+FROM pg_buffercache
+GROUP BY 1, 2`
+
+func GetBufferCache(ctx context.Context, server *state.Server, globalCollectionOpts state.CollectionOpts, logger *util.Logger, postgresVersion state.PostgresVersion, channel chan state.BufferCache) {
```

Comment:
Currently this doesn't perform a Postgres version check or include a way to disable the functionality. Thoughts?